.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+    'env': {
+        'browser': true,
```

Comment:
Is ``browser`` needed? This should be a node-only package.

Reply:
Yeah, ah... missed that. I though I've passed "node" as preset to eslint --init, but then it put browser from inspect of existing code. Blah